tests/test_grids.py

```diff
@@ -559,7 +559,7 @@ def test_popgrid(mode, vert_discretisation):
                  'V': 'V',
                  'W': 'W',
                  'T': 'T'}
-    dimensions = {'lon': 'lon', 'lat': 'lat', 'depth': w_dep}
+    dimensions = {'lon': 'lon', 'lat': 'lat', 'depth': w_dep, 'time': 'time'}
```

Comment:
So are we sure that this now tests for `deferred_load=True`? Should we add that as a pytest option (`@pytest.mark.parametrize('deferred_load', [True, False])`)?

Reply:
Yes, I would do that. @pdnooteboom, you can add it as a flag to `from_pop`